src/platform/silabs/platformAbstraction/WiseMcuSpam.cpp

```diff
@@ -96,10 +95,10 @@ CHIP_ERROR SilabsPlatform::Init(void)
     // TODO: Setting the highest priority for SVCall_IRQn to avoid the HardFault issue
     NVIC_SetPriority(SVCall_IRQn, CORE_INTERRUPT_HIGHEST_PRIORITY);
 
-#if CHIP_CONFIG_ENABLE_ICD_SERVER == 0
+#if (CHIP_CONFIG_ENABLE_ICD_SERVER == 0) && (SL_PROVISION_GENERATOR == 0)
```

Comment:
Correct me if I am wrong but SL_PROVISION_GENERATOR is never defined when it isn't the GFW. This could lead to an undefined value being used.

maybe 
`#if defined(CHIP_CONFIG_ENABLE_ICD_SERVER) && (CHIP_CONFIG_ENABLE_ICD_SERVER == 0) `
would do the trick. I don't think CHIP_CONFIG_ENABLE_ICD_SERVER us defined in gfw